### ISSUE TITLE
Bulk insert not working?

### ISSUE THREAD

**Issue by chusband** [Toblerity/rtree#68]
Using version 1.8.5 of libspatialindex and version 0.8.2 of rtree

```python
from rtree import index
from random import random
def generator_function():
    for i, coord in enumerate((random(), random()) for i in range(10000)):
        yield (i, (coord[0], coord[1], coord[0], coord[1]), None)
%time tree = index.Index(generator_function())
```
RTreeError: Error in "Index_CreateWithStream": b'IllegalArgumentException: Region::combineRegion: Region has different number of dimensions.'

**Comment by soumyami** [comment]
More details will be required to see what the problem is, but I would suggest using a different version of rtree.

**Comment by chusband** [comment]
version 0.8.0 seems to fix it.

Not sure what else you want.  We got the version of the IOOS channel in conda, so maybe that could help.

**Comment by soumyami** [comment]
The problem I encountered was on RHEL, and not Debian. And yes, 0.8.0 works well for bulk insert. Glad you solved it.

**Comment by edwinwid** [comment]
Hi, facing the same problem. However, downgrading rtree version may not be an option for me. Did a quick scan of the .py, but I don't see any obvious change from 0.8.0 to 0.8.2 that would have cause this... as anyone looked deeper?

**Comment by hobu** [comment]
I have spent some time on this issue, but I have not gotten to a satisfactory conclusion. If I build libspatialindex in debug mode, bulk insertion works, but if I build it in release mode, it does not. Can you try that scenario and see if it "fixes" it for you too?

**Comment by hobu** [comment]
Fixed in #77
